Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -199,56 +188,36 @@ - (BOOL)setVideoBitrate:(int)bitrate forCall:(OCTCall *)call error:(NSError **)e
 }
 
 #pragma mark Private methods
-- (OCTCall *)callFromFriend:(OCTToxFriendNumber)friendNumber
+- (OCTCall *)getOrCreateCallWithFriend:(OCTToxFriendNumber)friendNumber
 {
-    OCTDBManager *dbManager = [self.dataSource managerGetDBManager];
-    OCTDBChat *chatDB = [dbManager getOrCreateChatWithFriendNumber:friendNumber];
-    OCTChat *chat = [self.chatConverter objectFromRLMObject:chatDB];
-
-    OCTCall *call = [[OCTCall alloc] initCallWithChat:chat];
-
-    return call;
+    return [[self.dataSource managerGetRealmManager] getOrCreateCallWithFriendNumber:friendNumber];
```

Comment:
I mean use getOrCreateCall... where you need it instead of wrapping it into method